src/main/resources/lang/english.lang

```diff
@@ -1265,6 +1265,16 @@ heal reasons:
 	wither: withering, withered, wither potion, a wither effect, wither effect, wither
 	wither_spawn: wither spawning, wither spawn, wither summoning, a wither summoning, a wither spawn, a wither spawning
 
+# -- Fishing States --
+fishing states:
```

Comment:
Maybe we could introduce things like `a failed attempt` for example. So users could do `if the fishing state is a failed attempt` - Not sure if that is already possible or not. Might not be much of an improvement either, so this one is up for discussion :D